package.json

```diff
@@ -86,7 +86,7 @@
     "react-native-permissions": "^3.0.1",
     "react-native-picker-select": "8.0.4",
     "react-native-plaid-link-sdk": "^7.0.5",
-    "react-native-reanimated": "^2.1.0",
+    "react-native-reanimated": "^2.3.0-alpha.1",
```

Comment:
Did you just do `npm i react-native-reanimated@next` ? Latest release on GH looks like `2.2.0`.

Reply:
I picked the most stable from the website https://docs.swmansion.com/react-native-reanimated/docs